### PR TITLE
feat(cli): convert ASTNode to string since now providers return raw s…

### PR DESCRIPTION
**Pull request by m-pizarro**
…chemas instead of strings

## Issue tracker links

_Add links to any relevant tasks/stories/bugs/pagerduty/etc_

*Example - dummy TODO project*

[TODO-123](https://autoclouddev.atlassian.net/browse/TODO-123)

## Changes/solution

_How does this change address the problem?_

## Testing

_Describe how the testing was done, plus evidence, if not covered by automated tests_

## Notes and considerations

_Add any additional notes and/or considerations_

## Dependencies

_Add dependencies on any other PRs, if applicable

### CODE REVIEW

File: src/commands/scan.ts [hunk 1]
```diff
@@ -217,7 +218,7 @@ export default class Scan extends Command {
           provider
         )}`
       )
-      const providerSchema: string = providerClient.getSchema()
+      const providerSchema: string = print(providerClient.getSchema())
```
Comment (tyler-dunkel) [inline src/commands/scan.ts:221]: what will happen with lower versioned providers?

Comment (m-pizarro) [reply src/commands/scan.ts:221]: oh ok, this method is called by the gqlapi and cli tool. The print() method was moved to those apps, I considered that for previous providers versions they should use the previous versions of these apps. But let me know if I'm missing something.

Comment (tyler-dunkel) [reply src/commands/scan.ts:221]: hmm, I would add the required versions to the package.json so they know the provider and cli are incompatible versions and also add a way to notify them they should update the provider version if they hit an error regarding this

**Review by hjaraujof — APPROVED** [review]
LGTM

**Pull request merged**